src/common/main/kotlin/tech/thatgravyboat/skyblockapi/api/events/misc/SkyBlockApiRegisterCommandsEvent.kt

```diff
@@ -0,0 +1,5 @@
+package tech.thatgravyboat.skyblockapi.api.events.misc
+
+class SkyBlockApiRegisterCommandsEvent(
```

Comment:
Should be internal

Reply:
I dont want anyone to even have a chance of using the wrong event so we shouldnt use it in sbapi itself then